guide/fde: reword/fix drive number/organize (#535)
void-linux/void-docs: src/installation/guides/fde.md

# Full Disk Encryption

**Warning**: Your drive's block device and other information may be different,

> so make *certain* it is correct.

Reviewer: I think the addition of a bold warning is ok, but please use "make sure" instead of "make certain".

## Partitioning

> Boot a live image and then login. Create a single physical partition on the disk

Reviewer: ```suggestion
Boot a live image and login.

Create a single physical partition on the disk
```

I kind of liked the split...

> using `cfdisk`, marking it as bootable. For an MBR system, the partition layout

Reviewer: cfdisk could become a link to the cfdisk manpage in man.voidlinux.org

# Full Disk Encryption

> **Warning**: Your drive's block device and other information may be different,
> so make *sure* it is correct.

Reviewer: I don't think double emphasis is necessary.

```suggestion
**Warning**: Your drive's block device and other information may be different,
so make sure it is correct.
```

/dev/sda1  *     2048 100663295 100661248  48G 83 Linux
```

> UEFI systems will need the disk to have both a GPT disklabel and an EFI system

Reviewer: Not sure this makes the text any different.

/dev/sda2  264192 100663262 100399071 47.9G Linux filesystem
```

> `cryptsetup` defaults to LUKS2, yet GRUB currently only has support for LUKS1,
> so it is critical to force LUKS1.
> 
> ## Encrypted volume configuration

Reviewer: ```suggestion
## Encrypted volume configuration

`cryptsetup` defaults to LUKS2, yet GRUB currently only has support for LUKS1,
so it is critical to force LUKS1.
```

cryptsetup should likely become a man page link as well.

> Keep in mind the encrypted volume will be `/dev/sda2` on EFI systems, as the EFI
> partition is `/dev/sda1`.

Reviewer: ```suggestion
Keep in mind the encrypted volume will be `/dev/sda2` on EFI systems, since `/dev/sda1` is taken up by the EFI partition.
```

```

> # cryptsetup luksFormat --type luks1 /dev/sda2

Reviewer: This should use `sda1`. We give a warning about it being `sda2` in the paragraph right above this, people should read these in order. Someone doing a BIOS setup can be confused.

/dev/sda2  264192 100663262 100399071 47.9G Linux filesystem
```

> [Cryptsetup](https://man.voidlinux.org/cryptsetup) defaults to LUKS2, yet GRUB currently only has support for LUKS1,
> so it is critical to force LUKS1.
> 
> ## Encrypted volume configuration

Reviewer: ```suggestion
## Encrypted volume configuration

[cryptsetup(8)](https://man.voidlinux.org/cryptsetup.8) defaults to LUKS2, yet GRUB currently only has support for LUKS1,
so it is critical to force LUKS1.
```

/dev/sda1	/boot/efi	vfat	defaults	0	0
```

> ### Grub configuration

Reviewer: ```suggestion
### GRUB configuration
```

for the `sda1` device found in the output of the
[blkid(8)](https://man.voidlinux.org/blkid.8) command above.

> ## Key setup

Reviewer: More descriptive title? This feels a bit generic.

install_items+=" /boot/volume.key /etc/crypttab "
```

> ## Finish installation

Reviewer: If this must be a header, maybe "Last installation steps" or "Complete system installation"
